blocks/v2-columns/v2-columns.js

```diff
@@ -0,0 +1,31 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+import { getTextLabel, createElement } from '../../scripts/common.js';
+
+const ctaText = getTextLabel('learn more');
+
+export default async function decorate(block) {
+  const picture = block.querySelector('picture');
+  const blockConfig = readBlockConfig(block);
+  const [, eyebrow, headline, body, link] = Object.values(blockConfig);
+
+  const columnsImage = createElement('div', { classes: 'v2-columns-image' });
+  const columnsText = createElement('div', { classes: 'v2-columns-content' });
+
+  const eyebrowElmt = createElement('p', { classes: 'v2-columns-content__eyebrow' });
+  eyebrowElmt.textContent = eyebrow;
+  const headlineElmt = createElement('h2', { classes: 'v2-columns-content__headline' });
```

Comment:
Why not letting the editor decide which tag to use? 🤔 
Not sure what is expected here, can you clarify? @cogniSyb @Lakshmishri

Reply:
The editor should use the tag he wants. But we could do the styling (fonts) in css